pvlib/irradiance.py

```diff
@@ -793,20 +790,30 @@ def haydavies(surface_tilt, surface_azimuth, dhi, dni, dni_extra,
 
     Notes
     ------
+    The expression for the diffuse irradiance, :math:`I_d` in the Hay and
+    Davies sky diffuse model is as follows:
+
+    .. math::
+            I_{d} = DHI ( A\cdot R_b + (1 - A) (\frac{1 + \cos\theta_T}{2}) ).
+
+    :math:`DHI` is the diffuse horizontal irradiance,
+    :math:`A` is the anisotropy index, which is the ratio of the direct normal
+    irradiance to the extraterrestrial irradiation,
+    :math:`R_b` is the projection ratio, which is defined as the cosine of the
+    ratio of angle of incidence (AOI) to the cosine of the zenith angle, and
+    finally :math:`\theta_T` is the tilt angle of the array.
+
     When supplying ``projection_ratio``, consider constraining its values
-    when zenith angle approaches 90 degrees or angle of incidence
+    when the zenith angle approaches 90 degrees or the angle of incidence
     projection is negative. See code for details.
 
     References
     -----------
-    .. [1] Loutzenhiser P.G. et. al. "Empirical validation of models to
-       compute solar irradiance on inclined surfaces for building energy
-       simulation" 2007, Solar Energy vol. 81. pp. 254-267
-
-    .. [2] Hay, J.E., Davies, J.A., 1980. Calculations of the solar
+    .. [1] Hay, J. E., Davies, J.A., 1980. Calculations of the solar
```

Comment:
Why are we dropping [1]? It was used here because it is easy to find (the 1980 Hay/Davies paper is not) and it contains a concise description of the Hay/Davies model.

Reply:
Ah okay I understand why it's there now. Since the [1] was not attributed anywhere in the docstring, I removed it. I have added it back in with an in-text reference ([3a9b965](https://github.com/pvlib/pvlib-python/pull/2191/commits/3a9b9659b5a241071a1dbdc10e878c887a796807)). Thanks for the suggestion